cryosparc/mrc.py

```diff
@@ -128,8 +128,12 @@ def write(file: Union[str, PurePath, IO[bytes]], data: "NDArray", psize: float):
     assert data.ndim == 3, "Cannot write an array in MRC file"
 
     with bopen(file, "wb") as f:
+        if data.dtype.type == DT.FLOAT16:
```

Comment:
`DT` is an enum we've defined in this file that represents the numeric data type identified that's written into the MRC header; these constants (e.g., `12` for float16) are unrelated to how these types are represented in numpy.

I think you mean to do this:

```suggestion
        if data.dtype.type == n.float16:
```

Reply:
Ah! I misread `DT_TO_DATATYPE`. Fixed in latest!